sdk/python/kfp/cli/install.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""CLI interface for KFP installation tool."""
+
+import json as json_library
+import sys
+from typing import Dict, Text
+import click
+from .install_cli import prerequest, project_id_resolver, kfp_installer, cluster_resolver, gcs_resolver, kfp_input_resolver, cloud_sql_resolver, gpu_resolver
+from ..__init__ import __version__
+
+@click.group()
+def install():
+  pass
+
+
+@install.command()
+@click.option(
+    '--gcp-project-id',
+    type=Text,
+    help='Target project id. It will use environment default if not specified.')
+@click.option(
+    '--gcp-cluster-id',
+    type=Text,
+    help='Namespace to use for Kubernetes cluster.')
+@click.option(
+    '--gcp-cluster-zone',
+    type=Text,
+    help='Namespace to use for Kubernetes cluster.')
+@click.option(
+    '--gcs-default-bucket',
+    type=Text,
+    help='GCS default bucket. No "gs://" prefix')
+@click.option(
+    '--instance-name',
+    type=Text,
+    help='The instance name of the Kubeflow Pipelines installation')
+@click.option(
+    '--namespace',
+    type=Text,
+    help='Namespace to use for Kubernetes cluster.')
+@click.option(
+    '--enable-managed-storage',
+    type=click.Choice(['true','false']),
+    help='Whether use CloudSQL & GCS to store KFP system data.'
+       + ' If false, it will use in-cluster volume while will be deleted in uninstallation.')
+@click.option(
+    '--cloud-sql-instance-name',
+    type=Text,
+    help='CloudSQL instance name formatted as projectId::region::name.')
+@click.option(
+    '--cloud-sql-username',
+    type=Text,
+    help='CloudSQL username.')
+@click.option(
+    '--cloud-sql-password',
+    type=Text,
+    help="CloudSQL password. Not suggested to pass password here.")
+@click.option(
+    '--install-version',
+    type=Text,
+    help="Install which version, latest or something like 1.0.0")
+@click.option(
+    '--keep-kustomize-directory',
+    is_flag=True,
+    help="Whether keep the generated kustomize directory for debug purpose")
```

Comment:
discussion: as discussed in the chat, I'd prefer we get rid of this argument and always keep the kustomize directory.
Also, the directory should be saved in CWD

Reply:
The CloudSQL password is in the kustomize directory.

Reply:
Can you explain why that's a concern?
If that's really a problem? Shall we consider simply not putting cloudsql password in the directory?

Alternative we can consider is just use `kubectl create secret` to create a secret with cloudsql password. We can have a conventional secret name, so other parts of the manifest can refer to that secret instead.